RealifeTech-CoreSDK/GraphQL/API/Types.graphql.swift

```diff
@@ -434,4 +449,64 @@ public enum ApolloType {
       ]
     }
   }
+
+  public enum WebPageType: RawRepresentable, Equatable, Hashable, CaseIterable, Apollo.JSONDecodable, Apollo.JSONEncodable {
+    public typealias RawValue = String
+    case audioGuidesHelp
+    case tAndC
+    case privacy
+    case about
+    case aboutCompany
+    case purchasesHelp
+    /// Auto generated constant for unknown enum values
+    case __unknown(RawValue)
+
+    public init?(rawValue: RawValue) {
+      switch rawValue {
+        case "audioGuidesHelp": self = .audioGuidesHelp
+        case "tAndC": self = .tAndC
+        case "privacy": self = .privacy
+        case "about": self = .about
+        case "aboutCompany": self = .aboutCompany
+        case "purchasesHelp": self = .purchasesHelp
+        default: self = .__unknown(rawValue)
+      }
+    }
+
+    public var rawValue: RawValue {
+      switch self {
+        case .audioGuidesHelp: return "audioGuidesHelp"
+        case .tAndC: return "tAndC"
+        case .privacy: return "privacy"
+        case .about: return "about"
+        case .aboutCompany: return "aboutCompany"
+        case .purchasesHelp: return "purchasesHelp"
+        case .__unknown(let value): return value
+      }
+    }
+
+    public static func == (lhs: WebPageType, rhs: WebPageType) -> Bool {
+      switch (lhs, rhs) {
+        case (.audioGuidesHelp, .audioGuidesHelp): return true
```

Comment:
Could these cases be comma separated:
```
public static func == (lhs: WebPageType, rhs: WebPageType) -> Bool {
      switch (lhs, rhs) {
        case (.audioGuidesHelp, .audioGuidesHelp),
                 (.tAndC, .tAndC),
                 (.privacy, .privacy),
                 (.about, .about),
                 (.aboutCompany, .aboutCompany),
                 (.purchasesHelp, .purchasesHelp): return true
        (.__unknown(let lhsValue), .__unknown(let rhsValue)): return lhsValue == rhsValue
        default: return false
      }
    }
```

Reply:
This file is generated automatically by Apollo